device/isis/multitopology.yaml

```diff
@@ -0,0 +1,21 @@
+components:
+  schemas:
+    Isis.MT:
+      x-status: under-review
+      description: |-
+        Container of properties per interface per topology when multiple topology is configured
```

Comment:
Container of properties per interface per topology when multiple topologies are configured in an ISIS router.